cmd/mirror-main.go

```diff
@@ -554,12 +554,8 @@ func (mj *mirrorJob) monitorMirrorStatus(cancel context.CancelFunc) (errDuringMi
 						ignoreErr = true
 					}
 					if !ignoreErr {
-						if !mj.opts.skipErrors {
```

Comment:
I'm no expert on `mc`, but it looks like it will now also log the error when not running with `--json` format. I guess it should be:
```go
if globalJSON || !mj.opts.skipErrors {
  errorIf(sURLs.Error.Trace(sURLs.SourceContent.URL.String()),
          "Failed to copy `%s`.", sURLs.SourceContent.URL)
}
```

Reply:
I think it should be printed as an error in either case.

Reply:
yes @ramondeklein, this error was always printing, whether --json is passed or not